python/felis/datamodel.py

```diff
@@ -225,6 +225,31 @@ class Column(BaseObject):
     votable_datatype: str | None = Field(None, alias="votable:datatype")
     """The VOTable datatype of the column."""
 
+    @model_validator(mode="after")
+    def check_value(self) -> Column:
+        """Check that the default value is valid."""
+        if (value := self.value) is not None:
+            if value is not None and self.autoincrement is True:
+                raise ValueError("Column cannot have both a default value and be autoincremented")
+            felis_type = FelisType.felis_type(self.datatype)
+            if felis_type.is_numeric:
+                if felis_type.felis_name in ("byte", "short", "int", "long"):
+                    if not isinstance(value, int):
```

Comment:
If you are worried that a numpy int might turn up then you need to be checking for `numbers.Integral` instead (and then casting to a real `int`).

Reply:
> If you are worried that a numpy int might turn up then you need to be checking for `numbers.Integral` instead (and then casting to a real `int`).

These should only ever be `int` or `float` in the `value` field coming from the YAML data. 

This field is not representing the actual column data. It is the setting for the default on the column in the database when generating DDL.